docs/serverless-v2/08-03-bind-service-instance.md

```diff
@@ -0,0 +1,148 @@
+    metadata:
+      name: $NAME
+      namespace: $NAMESPACE
+    spec:
+      serviceClassExternalName: redis
+      servicePlanExternalName: micro
+      parameters:
+        imagePullPolicy: Always
+    EOF    
+    ```
+
+5. Check if the ServiceInstance CR was created successfully. The last condition in the CR status should state `Ready True`:
+
+    ```bash
+    kubectl get serviceinstance $NAME -n $NAMESPACE -o=jsonpath="{range .status.conditions[*]}{.type}{'\t'}{.status}{'\n'}{end}"
+    ```
+
+6. Create a ServiceBinding CR that points to the newly created Service Instance in the **spec.instanceRef** field:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: servicecatalog.k8s.io/v1beta1
+    kind: ServiceBinding
+    metadata:
+      name: $NAME
+      namespace: $NAMESPACE
+    spec:
+      instanceRef:
+        name: $NAME
+    EOF    
+    ```
+
+    > **NOTE:** If you use an existing Service Instance, change **spec.instanceRef.name** to the name of your Service Instance.
+
+7. Check if the ServiceBinding CR was created successfully. The last condition in the CR status should state `Ready True`:
+
+    ```bash
+    kubectl get servicebinding $NAME -n $NAMESPACE -o=jsonpath="{range .status.conditions[*]}{.type}{'\t'}{.status}{'\n'}{end}"
+    ```
+
+8. Create a ServiceBindingUsage CR:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: servicecatalog.kyma-project.io/v1alpha1
+    kind: ServiceBindingUsage
+    metadata:
+      name: $NAME
+      namespace: $NAMESPACE
+    spec:
+      serviceBindingRef:
+        name: $NAME
+      usedBy:
+        kind: knative-service
+        name: $NAME
+      parameters:
+        envPrefix:
+          name: "REDIS_"
+    EOF    
+    ```
+
+    - The **spec.serviceBindingRef** and **spec.usedBy** fields are required. **spec.serviceBindingRef** points to the Service Binding you have just created and **spec.usedBy** points to the lambda. More specifically, **spec.usedBy** refers to the name of the related KService CR (`name: $NAME`) and the cluster-specific [UsageKind CR](https://kyma-project.io/docs/components/service-catalog/#custom-resource-usage-kind) (`kind: knative-service`) that defines how Secrets should be injected to your lambda through the Service Binding.
+
+    > **NOTE:** Read [this](/components/service-catalog/#details-provisioning-and-binding) document to learn more about binding in Kyma.
+
+    - The **spec.parameters.envPrefix.name** field is optional. It adds a prefix to all environment variables injected by a given Secret from the Service Binding to the lambda. In our example, **envPrefix** is `REDIS_`, so all environmental variables will follow the `REDIS_{env}` naming pattern.
+
+    > **TIP:** It is considered good practice to use **envPrefix**. In some cases, lambda must use several instances of a given Service Class, so prefixes allow you to distinguish between instances and make sure that one Secret does not overwrite another one.
+
```

Comment:
Could you adjust indents?

![image](https://user-images.githubusercontent.com/40655785/75853853-31113680-5def-11ea-8ba9-fca3e6fb9c31.png)